### ISSUE TITLE
App configuration

### ISSUE THREAD

**Issue by medrockstar** [STS-Rosario/carpoolear#43]
Hi,

how we can configure phonegap app ? to get the correct URL

**Comment by p4bl1t0** [comment]
In the [dev.env.js](https://github.com/STS-Rosario/carpoolear/blob/master/config/dev.env.js) or the [prod.env.js](https://github.com/STS-Rosario/carpoolear/blob/master/config/prod.env.js) depends on the kind of build your are trying to do.

**Comment by medrockstar** [comment]
I have many error like this :
/api/trips?is_passenger=false&page=1&page_size=20 500 (Internal Server Error)
/api/users/my-trips?as_driver=false 500 (Internal Server Error)
/api/users/requests 500 (Internal Server Error)

**Comment by p4bl1t0** [comment]
Setting the API_URL

https://github.com/STS-Rosario/carpoolear/blob/master/config/dev.env.js#L6

**Comment by medrockstar** [comment]
yes this is what I did before, but I think the mobile app can't connect to the localhost.

**Comment by p4bl1t0** [comment]
Never a mobile phone could connect to a localhost address, you have to publish using a local IP or a service like nip.io

**Comment by carlosabud** [comment]
Assuming this is not a real issue then?
